src/runtime/driver/mod.rs

```diff
@@ -56,7 +57,7 @@ impl Driver {
         self.ops.lifecycle.len()
     }
 
-    pub(crate) fn tick(&mut self) {
+    pub(crate) fn dispatch_completions(&mut self) {
```

Comment:
True the name `tick` didn't do this justice since it was not controlled by a clock. How about something without the name completions in it? Complete and completable are all over the place already. Could you go with `cq_sync`? Or even `cq_sync_and_dispatch`?

Reply:
I feel like this name does a good job of describing what we are doing here. We should probably change some other methods though. You are right about there being too many things with `complete` in the name.